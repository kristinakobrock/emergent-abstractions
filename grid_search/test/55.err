  0%|                                                                                                     | 0/50 [00:00<?, ?it/s]  6%|#####5                                                                                       | 3/50 [00:00<00:01, 23.70it/s] 12%|###########1                                                                                 | 6/50 [00:00<00:01, 22.21it/s] 18%|################7                                                                            | 9/50 [00:00<00:01, 23.45it/s] 26%|#######################9                                                                    | 13/50 [00:00<00:01, 25.78it/s] 32%|#############################4                                                              | 16/50 [00:00<00:01, 24.61it/s] 38%|##################################9                                                         | 19/50 [00:00<00:01, 25.28it/s] 44%|########################################4                                                   | 22/50 [00:00<00:01, 24.29it/s] 54%|#################################################6                                          | 27/50 [00:01<00:00, 29.83it/s] 62%|#########################################################                                   | 31/50 [00:01<00:00, 29.90it/s] 70%|################################################################3                           | 35/50 [00:01<00:00, 26.49it/s] 76%|#####################################################################9                      | 38/50 [00:01<00:00, 27.11it/s] 82%|###########################################################################4                | 41/50 [00:01<00:00, 25.60it/s] 88%|################################################################################9           | 44/50 [00:01<00:00, 25.43it/s] 94%|######################################################################################4     | 47/50 [00:01<00:00, 25.43it/s]100%|############################################################################################| 50/50 [00:01<00:00, 23.53it/s]100%|############################################################################################| 50/50 [00:01<00:00, 25.41it/s]
  0%|                                                                                                     | 0/13 [00:00<?, ?it/s] 23%|#####################4                                                                       | 3/13 [00:00<00:00, 25.31it/s] 46%|##########################################9                                                  | 6/13 [00:00<00:00, 25.00it/s] 69%|################################################################3                            | 9/13 [00:00<00:00, 26.73it/s] 92%|####################################################################################9       | 12/13 [00:00<00:00, 27.71it/s]100%|############################################################################################| 13/13 [00:00<00:00, 26.13it/s]
