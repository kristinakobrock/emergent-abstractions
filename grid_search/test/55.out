# ["--attributes=5", "--values=4", "--game_size=10", "--vocab_size_factor=3", "--n_epochs=60", "--batch_size=32", "--learning_rate=0.001", "--hidden_size=256", "--temp_update=0.99", "--temperature=2"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.6971248984336853, "acc": 0.5966682434082031, "length": 1.8258212804794312, "mode": "train", "epoch": 1}
{"loss": 0.6593198776245117, "acc": 0.6321181058883667, "length": 1.0, "mode": "test", "epoch": 1}
{"loss": 0.6623720526695251, "acc": 0.6221169233322144, "length": 1.015631914138794, "mode": "train", "epoch": 2}
{"loss": 0.6592580676078796, "acc": 0.629687488079071, "length": 1.0, "mode": "test", "epoch": 2}
{"loss": 0.6617483496665955, "acc": 0.6201695799827576, "length": 1.0119208097457886, "mode": "train", "epoch": 3}
{"loss": 0.6577129364013672, "acc": 0.6315972805023193, "length": 1.0, "mode": "test", "epoch": 3}
{"loss": 0.6619343161582947, "acc": 0.6200176477432251, "length": 1.01093327999115, "mode": "train", "epoch": 4}
{"loss": 0.6623192429542542, "acc": 0.6307291984558105, "length": 1.0, "mode": "test", "epoch": 4}
{"loss": 0.6606088280677795, "acc": 0.6215391755104065, "length": 1.0112552642822266, "mode": "train", "epoch": 5}
{"loss": 0.6584827899932861, "acc": 0.6326388120651245, "length": 1.0, "mode": "test", "epoch": 5}
{"loss": 0.6613514423370361, "acc": 0.6204344034194946, "length": 1.0111292600631714, "mode": "train", "epoch": 6}
{"loss": 0.6585765480995178, "acc": 0.6328124403953552, "length": 1.0, "mode": "test", "epoch": 6}
{"loss": 0.6601935029029846, "acc": 0.6215606331825256, "length": 1.0113909244537354, "mode": "train", "epoch": 7}
{"loss": 0.6605584621429443, "acc": 0.6302083730697632, "length": 1.0, "mode": "test", "epoch": 7}
{"loss": 0.6615389585494995, "acc": 0.6205435991287231, "length": 1.0113166570663452, "mode": "train", "epoch": 8}
{"loss": 0.6575445532798767, "acc": 0.6314235925674438, "length": 1.0, "mode": "test", "epoch": 8}
{"loss": 0.6612139344215393, "acc": 0.619790256023407, "length": 1.0102707147598267, "mode": "train", "epoch": 9}
{"loss": 0.6607226729393005, "acc": 0.629687488079071, "length": 1.0, "mode": "test", "epoch": 9}
{"loss": 0.6597747802734375, "acc": 0.62167888879776, "length": 1.0087144374847412, "mode": "train", "epoch": 10}
{"loss": 0.6641085743904114, "acc": 0.6321181058883667, "length": 1.0, "mode": "test", "epoch": 10}
{"loss": 0.6591022610664368, "acc": 0.6224101781845093, "length": 1.0063810348510742, "mode": "train", "epoch": 11}
{"loss": 0.6592152714729309, "acc": 0.6310763955116272, "length": 1.0, "mode": "test", "epoch": 11}
{"loss": 0.6618695855140686, "acc": 0.6209734678268433, "length": 1.005323052406311, "mode": "train", "epoch": 12}
{"loss": 0.6612542867660522, "acc": 0.6328125, "length": 1.0, "mode": "test", "epoch": 12}
{"loss": 0.6621403098106384, "acc": 0.6185400485992432, "length": 1.0047698020935059, "mode": "train", "epoch": 13}
{"loss": 0.6615899205207825, "acc": 0.6315972805023193, "length": 1.0, "mode": "test", "epoch": 13}
{"loss": 0.6588082313537598, "acc": 0.6231361627578735, "length": 1.004244327545166, "mode": "train", "epoch": 14}
{"loss": 0.6619491577148438, "acc": 0.6328125, "length": 1.0, "mode": "test", "epoch": 14}
{"loss": 0.6607255339622498, "acc": 0.6193034648895264, "length": 1.0036450624465942, "mode": "train", "epoch": 15}
{"loss": 0.6676690578460693, "acc": 0.6116319894790649, "length": 1.0, "mode": "test", "epoch": 15}
{"loss": 0.6608166098594666, "acc": 0.6205215454101562, "length": 1.0027939081192017, "mode": "train", "epoch": 16}
{"loss": 0.6604207158088684, "acc": 0.6324653029441833, "length": 1.0, "mode": "test", "epoch": 16}
{"loss": 0.658669650554657, "acc": 0.6219801306724548, "length": 1.003062129020691, "mode": "train", "epoch": 17}
{"loss": 0.6588586568832397, "acc": 0.631944477558136, "length": 1.0, "mode": "test", "epoch": 17}
{"loss": 0.6599313020706177, "acc": 0.6206428408622742, "length": 1.0025607347488403, "mode": "train", "epoch": 18}
{"loss": 0.6593354344367981, "acc": 0.629687488079071, "length": 1.0, "mode": "test", "epoch": 18}
{"loss": 0.6586237549781799, "acc": 0.6222648620605469, "length": 1.0024731159210205, "mode": "train", "epoch": 19}
{"loss": 0.6598272919654846, "acc": 0.6322916150093079, "length": 1.0, "mode": "test", "epoch": 19}
{"loss": 0.658630907535553, "acc": 0.6240637302398682, "length": 1.002077341079712, "mode": "train", "epoch": 20}
{"loss": 0.6602092385292053, "acc": 0.6303818821907043, "length": 1.0, "mode": "test", "epoch": 20}
{"loss": 0.6580836772918701, "acc": 0.6219817399978638, "length": 1.0020142793655396, "mode": "train", "epoch": 21}
{"loss": 0.6607838273048401, "acc": 0.628125011920929, "length": 1.0, "mode": "test", "epoch": 21}
{"loss": 0.657637894153595, "acc": 0.6215739846229553, "length": 1.001538872718811, "mode": "train", "epoch": 22}
{"loss": 0.661508321762085, "acc": 0.6291666030883789, "length": 1.0, "mode": "test", "epoch": 22}
{"loss": 0.6589089035987854, "acc": 0.6229028701782227, "length": 1.0015815496444702, "mode": "train", "epoch": 23}
{"loss": 0.6615646481513977, "acc": 0.6322916150093079, "length": 1.0, "mode": "test", "epoch": 23}
{"loss": 0.6582810878753662, "acc": 0.6214016079902649, "length": 1.0013004541397095, "mode": "train", "epoch": 24}
{"loss": 0.6598853468894958, "acc": 0.6335070133209229, "length": 1.0, "mode": "test", "epoch": 24}
{"loss": 0.658661425113678, "acc": 0.6226723790168762, "length": 1.0012633800506592, "mode": "train", "epoch": 25}
{"loss": 0.6618483662605286, "acc": 0.6256945133209229, "length": 1.0, "mode": "test", "epoch": 25}
{"loss": 0.658708930015564, "acc": 0.6228446960449219, "length": 1.0011522769927979, "mode": "train", "epoch": 26}
{"loss": 0.6614126563072205, "acc": 0.6322916150093079, "length": 1.0, "mode": "test", "epoch": 26}
{"loss": 0.6577141880989075, "acc": 0.6227927803993225, "length": 1.0010943412780762, "mode": "train", "epoch": 27}
{"loss": 0.6617189645767212, "acc": 0.6270833015441895, "length": 1.0, "mode": "test", "epoch": 27}
{"loss": 0.6568756103515625, "acc": 0.623665452003479, "length": 1.00106942653656, "mode": "train", "epoch": 28}
{"loss": 0.6617424488067627, "acc": 0.6326388120651245, "length": 1.0, "mode": "test", "epoch": 28}
{"loss": 0.6581339836120605, "acc": 0.6226767897605896, "length": 1.001055121421814, "mode": "train", "epoch": 29}
{"loss": 0.6610745191574097, "acc": 0.6249999403953552, "length": 1.0, "mode": "test", "epoch": 29}
{"loss": 0.6578798890113831, "acc": 0.6230233907699585, "length": 1.0009702444076538, "mode": "train", "epoch": 30}
{"loss": 0.6610206961631775, "acc": 0.6321181654930115, "length": 1.0, "mode": "test", "epoch": 30}
{"loss": 0.6576690673828125, "acc": 0.6227360963821411, "length": 1.0009069442749023, "mode": "train", "epoch": 31}
{"loss": 0.6611372828483582, "acc": 0.6302083134651184, "length": 1.0, "mode": "test", "epoch": 31}
{"loss": 0.6580123901367188, "acc": 0.6245923042297363, "length": 1.0008500814437866, "mode": "train", "epoch": 32}
{"loss": 0.6595067977905273, "acc": 0.6310763955116272, "length": 1.0, "mode": "test", "epoch": 32}
{"loss": 0.657863974571228, "acc": 0.6233701109886169, "length": 1.0008500814437866, "mode": "train", "epoch": 33}
{"loss": 0.6601431965827942, "acc": 0.6295139789581299, "length": 1.0, "mode": "test", "epoch": 33}
{"loss": 0.6583461165428162, "acc": 0.6227928400039673, "length": 1.0006986856460571, "mode": "train", "epoch": 34}
{"loss": 0.6624277830123901, "acc": 0.6302082538604736, "length": 1.0, "mode": "test", "epoch": 34}
{"loss": 0.656963050365448, "acc": 0.6236650347709656, "length": 1.0009372234344482, "mode": "train", "epoch": 35}
{"loss": 0.6611660122871399, "acc": 0.6293402910232544, "length": 1.0, "mode": "test", "epoch": 35}
{"loss": 0.6573052406311035, "acc": 0.6248810291290283, "length": 1.0009397268295288, "mode": "train", "epoch": 36}
{"loss": 0.6644502878189087, "acc": 0.621874988079071, "length": 1.0, "mode": "test", "epoch": 36}
{"loss": 0.6578773260116577, "acc": 0.6236634850502014, "length": 1.0008316040039062, "mode": "train", "epoch": 37}
{"loss": 0.660515546798706, "acc": 0.6272569894790649, "length": 1.0, "mode": "test", "epoch": 37}
{"loss": 0.6578524112701416, "acc": 0.6236606240272522, "length": 1.0009381771087646, "mode": "train", "epoch": 38}
{"loss": 0.6618731021881104, "acc": 0.6324653029441833, "length": 1.0, "mode": "test", "epoch": 38}
{"loss": 0.657795250415802, "acc": 0.6234331130981445, "length": 1.001043438911438, "mode": "train", "epoch": 39}
{"loss": 0.6621209383010864, "acc": 0.6322916150093079, "length": 1.0, "mode": "test", "epoch": 39}
{"loss": 0.6574028730392456, "acc": 0.6240130066871643, "length": 1.0016469955444336, "mode": "train", "epoch": 40}
{"loss": 0.6612769961357117, "acc": 0.6295139789581299, "length": 1.0, "mode": "test", "epoch": 40}
{"loss": 0.6585151553153992, "acc": 0.6225476264953613, "length": 1.0066317319869995, "mode": "train", "epoch": 41}
{"loss": 0.6606959700584412, "acc": 0.6324653029441833, "length": 1.0, "mode": "test", "epoch": 41}
{"loss": 0.6411112546920776, "acc": 0.6435235142707825, "length": 1.5281906127929688, "mode": "train", "epoch": 42}
{"loss": 0.6500675678253174, "acc": 0.6236925721168518, "length": 1.2118055820465088, "mode": "test", "epoch": 42}
{"loss": 0.6144678592681885, "acc": 0.6633319854736328, "length": 1.495341420173645, "mode": "train", "epoch": 43}
{"loss": 0.6303541660308838, "acc": 0.6487196087837219, "length": 1.2430555820465088, "mode": "test", "epoch": 43}
{"loss": 0.605104386806488, "acc": 0.6716607809066772, "length": 1.4451364278793335, "mode": "train", "epoch": 44}
{"loss": 0.6186023354530334, "acc": 0.6448892951011658, "length": 1.21875, "mode": "test", "epoch": 44}
{"loss": 0.6051279902458191, "acc": 0.6727149486541748, "length": 1.311908483505249, "mode": "train", "epoch": 45}
{"loss": 0.6118422746658325, "acc": 0.6525228023529053, "length": 1.21875, "mode": "test", "epoch": 45}
{"loss": 0.6002269387245178, "acc": 0.6733040809631348, "length": 1.393212080001831, "mode": "train", "epoch": 46}
{"loss": 0.6078705191612244, "acc": 0.6681477427482605, "length": 1.2256944179534912, "mode": "test", "epoch": 46}
{"loss": 0.5964420437812805, "acc": 0.6753726005554199, "length": 1.2587497234344482, "mode": "train", "epoch": 47}
{"loss": 0.6070026159286499, "acc": 0.6564127206802368, "length": 1.2291666269302368, "mode": "test", "epoch": 47}
{"loss": 0.5976660847663879, "acc": 0.6724143624305725, "length": 1.2652567625045776, "mode": "train", "epoch": 48}
{"loss": 0.6097564101219177, "acc": 0.6520127654075623, "length": 1.2222222089767456, "mode": "test", "epoch": 48}
{"loss": 0.5967902541160583, "acc": 0.6750594973564148, "length": 1.2660284042358398, "mode": "train", "epoch": 49}
{"loss": 0.6182457804679871, "acc": 0.6556803584098816, "length": 1.2395833730697632, "mode": "test", "epoch": 49}
{"loss": 0.597699761390686, "acc": 0.677058756351471, "length": 1.2621910572052002, "mode": "train", "epoch": 50}
{"loss": 0.6178073883056641, "acc": 0.6562283039093018, "length": 1.2361111640930176, "mode": "test", "epoch": 50}
{"loss": 0.5954127311706543, "acc": 0.6764504909515381, "length": 1.2592014074325562, "mode": "train", "epoch": 51}
{"loss": 0.608883798122406, "acc": 0.6575033068656921, "length": 1.2256944179534912, "mode": "test", "epoch": 51}
{"loss": 0.5957038402557373, "acc": 0.6751512885093689, "length": 1.260704755783081, "mode": "train", "epoch": 52}
{"loss": 0.6120828986167908, "acc": 0.6532606482505798, "length": 1.2326388359069824, "mode": "test", "epoch": 52}
{"loss": 0.5951941013336182, "acc": 0.6753063797950745, "length": 1.267310380935669, "mode": "train", "epoch": 53}
{"loss": 0.6085265874862671, "acc": 0.6574327945709229, "length": 1.2291666269302368, "mode": "test", "epoch": 53}
{"loss": 0.5940761566162109, "acc": 0.6781644821166992, "length": 1.2632468938827515, "mode": "train", "epoch": 54}
{"loss": 0.6144272685050964, "acc": 0.649392306804657, "length": 1.2361111640930176, "mode": "test", "epoch": 54}
{"loss": 0.595574140548706, "acc": 0.676803708076477, "length": 1.2931101322174072, "mode": "train", "epoch": 55}
{"loss": 0.6123403906822205, "acc": 0.667285144329071, "length": 1.2326388359069824, "mode": "test", "epoch": 55}
{"loss": 0.5949162840843201, "acc": 0.6763787865638733, "length": 1.2567501068115234, "mode": "train", "epoch": 56}
{"loss": 0.6207401752471924, "acc": 0.6617838144302368, "length": 1.2395833730697632, "mode": "test", "epoch": 56}
{"loss": 0.5837639570236206, "acc": 0.6862273812294006, "length": 1.61362624168396, "mode": "train", "epoch": 57}
{"loss": 0.588616132736206, "acc": 0.6896267533302307, "length": 1.7395833730697632, "mode": "test", "epoch": 57}
{"loss": 0.569726288318634, "acc": 0.700045645236969, "length": 1.6821236610412598, "mode": "train", "epoch": 58}
{"loss": 0.5947449207305908, "acc": 0.6765299439430237, "length": 1.6840277910232544, "mode": "test", "epoch": 58}
{"loss": 0.5633761882781982, "acc": 0.7036222815513611, "length": 1.8396116495132446, "mode": "train", "epoch": 59}
{"loss": 0.6011654138565063, "acc": 0.6756889820098877, "length": 1.8958333730697632, "mode": "test", "epoch": 59}
{"loss": 0.552333652973175, "acc": 0.7108395099639893, "length": 2.059981346130371, "mode": "train", "epoch": 60}
{"loss": 0.5715517401695251, "acc": 0.6980903148651123, "length": 2.5520832538604736, "mode": "test", "epoch": 60}
test accuracy: 0.6855280995368958
